ad7768-1fmcz/spi_engine.c

```diff
@@ -439,8 +496,8 @@ int32_t spi_eng_transfer_multiple_msgs(spi_dev *dev, uint8_t no_of_messages)
 /***************************************************************************//**
 * @brief spi_eng_setup
 *******************************************************************************/
-int32_t spi_eng_setup(spi_dev **device,
```

Comment:
Should we have this modification as a separate commit since it relates to the spi_engine more?

Reply:
The latest version of spi_engine also uses `spi_init `instead of `spi_eng_setup`.